customer/pom.xml

```
@@ -15,11 +15,17 @@
 	<description>Customer micro-service</description>
 	<properties>
 		<java.version>17</java.version>
+		<org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
 		<sonar.organization>xandervedder</sonar.organization>
 		<sonar.host.url>https://sonarcloud.io/</sonar.host.url>
 		<sonar.exclusions>
 			**/CustomerApplication.java,
-			**/infrastructure/**
+			**/infrastructure/**,
+			**/application/command/**,
+			**/application/query/**,
+			**/core/domain/event/**,
+			**/application/*.*,
+			**/application/exception/*.*,
```

```suggestion
                        **/application/**,
                        **/domain/event/**,
                        **/infrastructure/**,
```

should do the trick, too